frontend/src/components/dashboard/components/services-item/services-item.tsx

```diff
@@ -1,5 +1,23 @@
 import { FC } from 'react';
+import { Button, Link } from 'components/common/common';
+import styles from './styles.module.scss';
+import { ServiceMenuItem } from 'common/types/ui/service-menu';
 
-const ServicesItem: FC = () => <article></article>;
+interface props {
+  service: ServiceMenuItem;
+}
 
+const ServicesItem: FC<props> = ({ service }) => (
+  <article className={styles.cardContent}>
+    <div className={styles.imgWrapper}>
+      <img src={service.img} alt={service.title} />
+    </div>
+    <span className={styles.cardTitle}>{service.title}</span>
+    <div className={styles.buttonContainer}>
+      <Link to={service.route}>
+        <Button label={'Start'} />
```

Comment:
button can't be in the `link`
you need to add `to` prop to `Button` component
And if it has `to` prop, we need to render a `Link` instead of `button`